src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -120,6 +121,12 @@ const RunTranscriptionView = ({ closeModal, nextView }: Props) => {
     dispatch,
   ]);
 
+  // we use !== instead of === since the useKeypress hook negates the condition that is given to it.
```

Comment:
If you have to explain it... :)

Reply:
i know... but if I fix the negation, then i'll have to explain two other usages of the `useKeypress` hook

Reply:
maybe the conditional could be renamed to something like `shouldEventHandlerRun` (I think @emmapainter mentioned this but I can't find it anywhere). Then no need to explain usage anywhere.

Regardless, I reckon developers should always be able to presume conditional is positive. If that means explaining it twice...

Reply:
I thought about it more and I changed my mind, here is the change: [56ed2a8](https://github.com/chloebrett/mlvet/pull/175/commits/56ed2a81f3a3e2dce6254eed2d987f8738609f8f)